### ISSUE TITLE
Settings and User Options broken

### ISSUE THREAD

**Issue by oceancolorcoder** [HDFGroup/hdfview#342]
Mac M3 Max
OS: Sonoma 14.7.1
Java: jdk 21.0.3
HDFView: 3.3.2

What I want to do:
Open HDFView>Settings
![Image](https://github.com/user-attachments/assets/dda4e320-1580-486b-9a40-16235bb489fa)

(Note also grayed out shortcuts)

What happens:
Nothing. No new windows open.

What I want to do:
Work with Tools>User Options>General Settings/HDF Settings/View Module Settings

What happens:
Preferences window opens with General Settings/HDF Settings/View Module Settings in the left column, but no content in the right column.
![Image](https://github.com/user-attachments/assets/8ef6ba44-9f55-4cf5-af68-41b15c7f64d9)

**Comment by DrJonnyT** [comment]
Looks like the same issue as #134 ?

**Comment by byrnHDF** [comment]
Yes, very likely - unfortunately I will need help from a mac developer on this - needs to be debugged on a real machine. Of course if you can enable logging with TRACE, maybe I could find the issue.

**Comment by byrnHDF** [comment]
As this is a new version of the Mac, then when 3.3.2 was released maybe getting a new SWT jar file and replacing it in the HDFView install location?
Start at https://eclipse.dev/eclipse/swt/

**Comment by byrnHDF** [comment]
I did notice that the jar files are in multiple locations. under /app and /app/mods

**Comment by byrnHDF** [comment]
Thanks for trying, I will need to locate a Mac for debugging.

**Comment by oceancolorcoder** [comment]
I did find the AARCH64 version and repeated the above with the same result.
https://mvnrepository.com/artifact/org.eclipse.platform/org.eclipse.swt.cocoa.macosx.aarch64/3.129.0

**Comment by oceancolorcoder** [comment]
Repeated the above for swt.jar in app/mods, and now the main window does not open.

**Comment by byrnHDF** [comment]
Wow, okay didn't expect that.
Will need to investigate.
